cmd/livepeer/starter/starter_test.go

```diff
@@ -109,6 +109,31 @@ func TestParseGetGatewayPrices(t *testing.T) {
 	}
 }
 
+func TestMaxPricePerCapability(t *testing.T) {
+	assert := assert.New(t)
+
+	jsonTemplate := `{"capabilities_prices": [ {"pipeline": "text-to-image", "model_id": "stabilityai/sd-turbo", "priceperunit": 1000, "pixelsperunit": 1}, {"pipeline": "image-to-video", "model_id": "default", "priceperunit": 2000, "pixelsperunit": 3} ] }`
+
+	prices := getCapabilityPrices(jsonTemplate)
+	assert.NotNil(prices)
+	assert.Equal(2, len(prices))
+
+	//confirm gateway field is initialized to empty string
+	assert.Equal(prices[0].Gateway, "")
```

Comment:
```suggestion
	assert.Equal(prices[0].Gateway, "default")
```
Update value for test

Reply:
Agree